### PR TITLE
KO: docs/site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb

### PR DESCRIPTION
**Pull request by YEERAMM**
KO : docs/site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb

translate raw_tensorflow_operators.ipynb
EN -> KO

EN : https://github.com/tensorflow/swift/blob/master/docs/site/tutorials/raw_tensorflow_operators.ipynb

### CODE REVIEW

**Comment by MarkDaoust** [comment]
Are you still working on this, or is it ready for a review?

**Comment by JKIsaacLee** [comment]
This pr contains 2 files.

**Comment by YEERAMM** [comment]
> Are you still working on this, or is it ready for a review?

ready for a review ! Is there any problem?

**Comment by YEERAMM** [comment]
> This pr contains 2 files.

Oh! Thanks :-) 
Thanks to you, I modified it.

**Comment by Choiuijin1125** [comment]
Hi @MarkDaoust  I start to review this PR, will finish asap :) thanks!!

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "yfNdITLmJtX8"
+   },
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://www.tensorflow.org/swift/tutorials/raw_tensorflow_operators\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
```
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: We translate 'View on TensorFlow.org' as 'TensorFlow.org에서 보기' :)
And please change the directory(code) for Ko. 'swift/tutorials' as 'swift/ko/tutorials'

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/swift/blob/master/docs/site/tutorials/raw_tensorflow_operators.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
```
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: 'Run in Google Colab' as '구글 코랩(Colab)에서 실행하기.
And please change the directory for Ko. 'docs/site/tutorials' as 'docs/site/ko/tutorials'

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/swift/blob/master/docs/site/tutorials/raw_tensorflow_operators.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
```
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: 'View source on GitHub' as '깃허브(GitHub) 소스 보기'
And please change the directory for Ko. 'docs/site/tutorials' as 'docs/site/ko/tutorials'

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "lONNcRalVUO9"
+   },
+   "source": [
```
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: We need to add declaration can you check other translated files? 
It start like this
'Note: 이 문서는 텐서플로 커뮤니티에서 번역했습니다. 커뮤니티 번역 활동의 특성상 정확한 번역과 최신 내용...'

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+    "# 다듬어지지 않은 텐서플로우 연산자들\n",
+    "\n",
+    "텐서플로우에서 빌드하는 것과, 텐서플로우의 Swift는 API design이 새로운 접근을 할 수 있게 합니다. API들은 확립된 라이브러리들로 부터 세밀하게 조직되어 있고, 새로운 관용구로 결합되어 있습니다. 이것은 모든 텐서플로우 API들이 직접적으로 Swift APIs로 사용되어질 수 있는 것은 아니라는 걸 뜻하고, 또 우리의 API curation이 상당한 시간과 발전 시키기 위한 헌신적인 노력이 필요하다는 것을 의미합니다. 만약 당신의 가장 좋아하는 텐서플로우 연산자가 Swift에서 사용 가능 하지 않는다고 해도 걱정하지 마세요. – 텐서플로우 Swift 라이브러리는 당신에게 Raw namespace 아래에서 대부분의 텐서플로우 연산자에 접근할 수 있도록 합니다..\n"
```
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: We translate tensorflow as '텐서플로' please check http://bit.ly/tf-docs-translation-status glossary sheet :)

And I think it's a bit mistranslated that 'Building on TensorFlow, Swift for TensorFlow takes a fresh approach to API design'

Building on + something is translated ~을 기반으로, 

I suggest 'Swift를 위한 텐서플로는 텐서플로를 기반으로 API 디자인에 새로운 접근을 할 수 있게 합니다.' or
'Swift를 위한 텐서플로는 API 디자인에 새로운 접근을 할 수 있게 합니다.'

And you can translate curation as '큐레이션' and 'Raw namespace' as '원시 네임스페이스'

**Comment by YEERAMM** [comment]
> Thank you for contribution. I'll review all asap :)

Thanks :-) Please let me know if you have anything else to modify.

**Comment by rickiepark** [comment]
Hi @YEERAMM , Thanks for contributing.
Whenever you push a commit, email notification is sent. :)
Please aggregate commits and push it at once.

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb [hunk 6]
```diff
@@ -0,0 +1,285 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "CYqNvcHxqg0Y"
+   },
+   "source": [
+    "`TensorFlow`를 시작하기 위해 불러오세요."
```
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: It's a bit awkward, why don't we just use '텐서플로를 시작하기 위해 불러오세요' ?

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb [hunk 7]
```diff
@@ -0,0 +1,285 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "5vza91sR09r-"
+   },
+   "source": [
+    "## 다듬어지지 않은 연산자 호출하기\n",
```
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: 'Raw 연산자 호출하기'

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb [hunk 8]
```diff
@@ -0,0 +1,285 @@
...
+    "\n",
+    "코드 완성기능을 통해 `원시` 네임스페이스에서 당신이 필요한 함수를 찾으세요."
```
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: Sorry for the double change '원시' → 'Raw'

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb [hunk 9]
```diff
@@ -0,0 +1,285 @@
...
+   "metadata": {
+    "colab_type": "text",
+    "id": "iIgKg-ueVCy_"
+   },
+   "source": [
+    "## 새로운 곱셈 연산자를 정의하기\n",
+    "\n",
+    "곱셈은 이미 `Tensor` 에서 `*` 연산자로 사용이 가능하지만, `.*`라는 새로운 이름의 연산자로 이용 가능 할 수 있도록 만들고 싶다고 가정해봅시다. Swift는 당신이 소급적으로 메소드를 추가할 수 있도록 하거나 `extension` 선언을 사용하는 타입들이 존재할 가능성을 계산할 수 있게 해줍니다.\n",
+    "\n",
+    "지금, `.*`를 extension을 선언하여 `Tensor`에 추가해봅시다. 그리고 tensor의 `Scalar` 타입이 [`Numeric`](https://developer.apple.com/documentation/swift/numeric)을 따를 때 이용가능 하도록 만드세요."
```
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: I'm feeling a bit awkward this sentence '지금, `.*`를 extension을 선언하여 `Tensor`에 추가해봅시다.'
what do u think? 
how about '지금, extension을 선언하여 `*` 연산자를 `Tensor`에 추가해봅시다.' ? 

And '이용가능 하도록 만드세요 → '이용 가능하도록 만드세요.'

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb [hunk 10]
```diff
@@ -0,0 +1,285 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "ucD5XZYYyzNe"
+   },
+   "source": [
+    "## 래퍼 함수에서 파생된 것을 정의하기\n",
+    "\n",
+    "당신은 쉽게 Swift API를 다듬어지지 않은 텐서플로우 연산자에서 정의할 수 있을 뿐만 아니라, Swift가 가지고 있는 최고 수준의 자동 미분계산 기능을 이용하여 다듬어지지 않은 텐서플로우 연산자를 미분할 수 있습니다.\n",
```
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: '텐서플로우' → '텐서플로'
first-class have a meaning on programming language(https://ko.wikipedia.org/wiki/%EC%9D%BC%EA%B8%89_%EA%B0%9D%EC%B2%B4)
So I suggest '최고 수준의 자동 미분계산 기능' → ' 일급 자동 미분 기능'

'다듬어지지 않은 텐서플로우 연산자' → 'Raw 텐서플로 연산자'

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb [hunk 11]
```diff
@@ -0,0 +1,285 @@
...
+    "        _ lhs: Tensor, _ rhs: Tensor\n",
+    "    ) -> (Tensor, (Tensor) -> (Tensor, Tensor)) {\n",
+    "        return (lhs * rhs, { v in\n",
+    "            ((rhs * v).unbroadcasted(to: lhs.shape),\n",
+    "            (lhs * v).unbroadcasted(to: rhs.shape))\n",
+    "        })\n",
+    "    }\n",
+    "}\n",
+    "\n",
+    "// Now, we can take the derivative of a function that calls `.*` that we just defined.\n",
```
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: It's better to translate this comment :)

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb (hunk 5 above)
Comment (Choiuijin1125) [reply site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: I just checked now Swift for TensorFlow translated 스위프트 포 텐서플로 can you change 'Swift를 위한 텐서플로는' → '스위프트 포 텐서플로'

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb [hunk 12]
```diff
@@ -0,0 +1,285 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "lONNcRalVUO9"
+   },
+   "source": [
+    "# 다듬어지지 않은 텐서플로우 연산자들\n",
+    "\n",
+    "Swift를 위한 텐서플로는 텐서플로를 기반으로 API 디자인에 새로운 접근을 할 수 있게 합니다. API들은 확립된 라이브러리들로 부터 세밀하게 조직되어 있고, 새로운 관용구로 결합되어 있습니다. 이것은 모든 텐서플로우 API들이 직접적으로 Swift APIs로 사용되어질 수 있는 것은 아니라는 걸 뜻하고, 또 우리의 API 큐레이션이 상당한 시간과 발전 시키기 위한 헌신적인 노력이 필요하다는 것을 의미합니다. 만약 당신의 가장 좋아하는 텐서플로우 연산자가 Swift에서 사용 가능 하지 않는다고 해도 걱정하지 마세요. – 텐서플로우 Swift 라이브러리는 당신에게 원시 네임스페이스 아래에서 대부분의 텐서플로우 연산자에 접근할 수 있도록 합니다.\n"
```
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: we translate Swift as '스위프트'(check PR 1161(https://github.com/tensorflow/docs/pull/1161))
So 'Swift' → '스위프트'

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb [hunk 13]
```diff
@@ -0,0 +1,285 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "iIgKg-ueVCy_"
+   },
+   "source": [
+    "## 새로운 곱셈 연산자를 정의하기\n",
+    "\n",
+    "곱셈은 이미 `Tensor` 에서 `*` 연산자로 사용이 가능하지만, `.*`라는 새로운 이름의 연산자로 이용 가능 할 수 있도록 만들고 싶다고 가정해봅시다. Swift는 당신이 소급적으로 메소드를 추가할 수 있도록 하거나 `extension` 선언을 사용하는 타입들이 존재할 가능성을 계산할 수 있게 해줍니다.\n",
```
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: 'Swift' → '스위프트'

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb (hunk 10 above)
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: 'Swift' → '스위프트'

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb [hunk 14]
```diff
@@ -0,0 +1,285 @@
...
+   "metadata": {
+    "colab_type": "text",
+    "id": "ucD5XZYYyzNe"
+   },
+   "source": [
+    "## 래퍼 함수에서 파생된 것을 정의하기\n",
+    "\n",
+    "당신은 쉽게 Swift API를 다듬어지지 않은 텐서플로우 연산자에서 정의할 수 있을 뿐만 아니라, Swift가 가지고 있는 최고 수준의 자동 미분계산 기능을 이용하여 다듬어지지 않은 텐서플로우 연산자를 미분할 수 있습니다.\n",
+    "\n",
+    "`.*`연산자를 미분 가능하게 만들기 위해서, `@differentiable` 속성을 이용하세요. 그리고 `vjp:` 레이블에서 고안된 함수를 속성 인자로써 구체화 시키세요. `.*`연산자가 제네릭 타입의 `Scalar`가 `Numeric`를 따르도록 정의되었기 때문에, `Tensor<Scalar>`가 `Differentiable` 프로토콜을 따르도록 만들기엔 충분하지(적절하지) 않습니다. 타입 세이프티를 지키는, Swift는 우리에게 `@differentiable` 속성의 `Scalar`가  `TensorFlowFloatingPoint` 프로토콜을 따르도록 요구하기 위해서, 제네릭 제약 조건을 추가하도록 요구할 것입니다. `TensorFlowFloatingPoint`는 `Tensor<Scalar>`가 `Differentiable`을 따르도록 만드는 프로토콜 입니다.\n",
```
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: 'Swift' → '스위프트'

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb (hunk 11 above)
Comment (YEERAMM) [reply site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: Is it correct to translate 'the deriavative of a function' into '함수의 도함수'?

**Comment by YEERAMM** [comment]
> Hi @YEERAMM , Thanks for contributing.
> Whenever you push a commit, email notification is sent. :)
> Please aggregate commits and push it at once.

Ok. Thank you for your advice. :-)

File: site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb [hunk 15]
```diff
@@ -0,0 +1,285 @@
...
+    "        _ lhs: Tensor, _ rhs: Tensor\n",
+    "    ) -> (Tensor, (Tensor) -> (Tensor, Tensor)) {\n",
+    "        return (lhs * rhs, { v in\n",
+    "            ((rhs * v).unbroadcasted(to: lhs.shape),\n",
+    "            (lhs * v).unbroadcasted(to: rhs.shape))\n",
+    "        })\n",
+    "    }\n",
+    "}\n",
+    "\n",
+    "// 이제, 우리는 방금 정의한 `.*`연산자를 호출하는 함수의 도함수를 구할 수 있습니다.\n",
```
Comment (Choiuijin1125) [inline site/ko/swift/docs/site/tutorials/raw_tensorflow_operators.ipynb]: '이제, 방금 정의한 ... 구할 수 있습니다.' :)

**Comment by Choiuijin1125** [comment]
Hi @lamberta :) This PR is LGTM 
thanks

**Comment by yashk2810** [comment]
Can you save the outputs for the swift notebooks?

**Pull request merged**